### ISSUE TITLE
New Implementation of categorical cross-entropy loss available in PyTorch

### ISSUE THREAD

**Issue by siebenkopf** [Vastlab/vast#61]
When I have lately checked the code of the `CrossEntropyLoss` in PyTorch, it seems that they now have an implementation of the categorical cross-entropy loss that allows to provide target vectors instead of only target classes: https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html
This is basically exactly what we are doing in our entropic open-set loss.

Hence, we should adapt our implementation of the categorical cross-entropy loss in our EOS loss: https://github.com/Vastlab/vast/blob/2ad7fa23cb8f0f1acb71d6b6d9aced6a70562e0c/vast/losses/losses.py#L133
and use the `CrossEntropyLoss` implementation provided by PyTorch instead.
